<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Apply 'actions' across multiple PyRQs</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>396</width>
           <height>449</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_Options">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Action:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_Actions">
               <item>
                <property name="text">
                 <string>Create</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Close</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Put</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Get</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>QSize</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MaxQSize</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_Blocking">
               <property name="text">
                <string>Blocking:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_Blocking">
               <item>
                <property name="text">
                 <string>Yes</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>No</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Surprise me!</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_Timeout">
               <property name="text">
                <string>Timeout:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBox_Timeout">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Custom value</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Random integer</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Unique random integer</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_QueueType">
               <property name="text">
                <string>Queue Type:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_QueueType">
               <item>
                <property name="text">
                 <string>List(synchronized)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Multiprocessing.Queue</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Don't care</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Surprise me!</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Unique per PyRQ</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_WhichQueues">
               <property name="text">
                <string>Which Queues:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBox_WhichQueues">
               <item>
                <property name="text">
                 <string>All</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A random queue</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>'n' random queues</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_Data">
               <property name="text">
                <string>Data:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QComboBox" name="comboBox_Data">
               <item>
                <property name="text">
                 <string>Surprise me!</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Randomly unique</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Custom value</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="1" column="0">
                 <widget class="QGroupBox" name="groupBox_Queue">
                  <property name="title">
                   <string>Queue options:</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QSpinBox" name="spinBox_CreateCount">
                     <property name="toolTip">
                      <string>How many queues to create on each PyRQ</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                     <property name="prefix">
                      <string>count </string>
                     </property>
                     <property name="minimum">
                      <number>-1</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QSpinBox" name="spinBox_CreateMaxsize">
                     <property name="toolTip">
                      <string>-1 = Not specified</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                     <property name="prefix">
                      <string>maxsize  </string>
                     </property>
                     <property name="minimum">
                      <number>-1</number>
                     </property>
                     <property name="maximum">
                      <number>100000000</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QGroupBox" name="groupBox_Timeout">
                  <property name="title">
                   <string>Timeout options:</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_TimeoutCustom">
                     <property name="text">
                      <string>Custom:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="spinBox_Timeout">
                     <property name="toolTip">
                      <string>timeout value for the action (seconds)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_TimeoutMax">
                     <property name="text">
                      <string>Max:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="spinBox_TimeoutMax">
                     <property name="toolTip">
                      <string>Max size for timeout</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <italic>true</italic>
                    <bold>true</bold>
                    <underline>true</underline>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>Scroll down for further options</string>
                  </property>
                  <property name="text">
                   <string>Options below...</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="9" column="0" colspan="2">
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Data options:</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLineEdit" name="lineEdit_Data">
                  <property name="toolTip">
                   <string>Enter text here to send in suitable actions (eg: PUT)</string>
                  </property>
                  <property name="text">
                   <string>enter text...</string>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_Randomise">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Randomise</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_Reset">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_Save">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Execute">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Add the actions to the script window.</string>
           </property>
           <property name="text">
            <string>Add to scripts...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
